extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -646,6 +646,45 @@ describe.each(['macos', 'windows'])('verify create on %s', os => {
   });
 });
 
+test('verify create command called in airgap mode will try to download hyperv image', async () => {
+  vi.mocked(extensionApi.env).isWindows = true;
```

Comment:
I don't get why isWindows is handled differently that isMac I think these are functions